### PR TITLE
Test E2E:  Adapt the 'GoUserStoryTest' according to new version of stack

### PR DESCRIPTION
**Pull request by artaleks9**
* Add change to the selenium test _GoUserStoryTest_ according to update version of the go-stack
* Related new version of _go-stack_: _quay.io/crw/stacks-golang:1.1-1_

### CODE REVIEW

**Comment by artaleks9** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test-okd

**Comment by SkorikSergey** [comment]
ci-build

**Comment by SkorikSergey** [comment]
ci-build

**Comment by SkorikSergey** [comment]
ci-test-okd

File: test/codeready-test-e2e/src/test/java/com/redhat/codeready/selenium/userstory/GoUserStoryTest.java [hunk 1]
```diff
@@ -116,7 +116,7 @@ public void checkCodeValidationFeature() {
     editor.typeTextIntoEditor("p");
     editor.waitMarkerInPosition(ERROR, 1);
     editor.moveToMarkerAndWaitAssistContent(ERROR);
-    editor.waitTextIntoAnnotationAssist("expected 'package', found 'IDENT' ppackage");
+    editor.waitTextIntoAnnotationAssist("expected 'package', found ppackage");
```
Comment (dmytro-ndp) [inline test/codeready-test-e2e/src/test/java/com/redhat/codeready/selenium/userstory/GoUserStoryTest.java]: It didn't found `'IDENT'` package. Are you sure it's normal?

Comment (dmytro-ndp) [reply test/codeready-test-e2e/src/test/java/com/redhat/codeready/selenium/userstory/GoUserStoryTest.java]: com redhat codeready selenium userstory GoUserStoryTest checkCodeValidationFeature_time-1554248995371-millis

Comment (artaleks9) [reply test/codeready-test-e2e/src/test/java/com/redhat/codeready/selenium/userstory/GoUserStoryTest.java]: I think yes

**Comment by SkorikSergey** [comment]
ci-test-okd

**Pull request merged**